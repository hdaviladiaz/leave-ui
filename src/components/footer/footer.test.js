import React from 'react';
import Footer from './footer';
import renderer from 'react-test-renderer';

it('should render Footer as snapshot', () => {
  const tree = renderer.create(<Footer/>).toJSON();
  expect(tree).toMatchSnapshot();
});
