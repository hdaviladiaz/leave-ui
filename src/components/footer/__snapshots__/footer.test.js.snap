// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer as snapshot 1`] = `
<div
  className="footer text-center"
>
  Â© Thoughtworks 2017
</div>
`;
